pkg/gadgets/top/tcp/types/types.go

```diff
@@ -44,18 +44,54 @@ func ParseFilterByFamily(family string) (int32, error) {
 type Stats struct {
 	eventtypes.CommonData
 
-	Saddr     string `json:"saddr,omitempty" column:"saddr"`
-	Daddr     string `json:"daddr,omitempty" column:"daddr"`
-	MountNsID uint64 `json:"mountnsid,omitempty" column:"mntns"`
-	Pid       int32  `json:"pid,omitempty" column:"pid"`
-	Comm      string `json:"comm,omitempty" column:"comm"`
-	Sport     uint16 `json:"sport,omitempty" column:"sport"`
-	Dport     uint16 `json:"dport,omitempty" column:"dport"`
-	Family    uint16 `json:"family,omitempty" column:"family"`
-	Sent      uint64 `json:"sent,omitempty" column:"sent"`
-	Received  uint64 `json:"received,omitempty" column:"received"`
+	MountNsID uint64 `json:"mountnsid,omitempty" column:"mntns,template:ns,hide"`
+	Pid       int32  `json:"pid,omitempty" column:"pid,template:pid"`
+	Comm      string `json:"comm,omitempty" column:"comm,template:comm"`
+	Family    uint16 `json:"family,omitempty" column:"ip,maxWidth:2"`
+	Saddr     string `json:"saddr,omitempty" column:"saddr,template:ipaddr,hide"`
+	Daddr     string `json:"daddr,omitempty" column:"daddr,template:ipaddr,hide"`
+	Sport     uint16 `json:"sport,omitempty" column:"sport,template:ipport,hide"`
+	Dport     uint16 `json:"dport,omitempty" column:"dport,template:ipport,hide"`
+	Sent      uint64 `json:"sent,omitempty" column:"sent,order:1002"`
+	Received  uint64 `json:"received,omitempty" column:"recv,order:1003"`
 }
 
 func GetColumns() *columns.Columns[Stats] {
-	return columns.MustCreateColumns[Stats]()
+	cols := columns.MustCreateColumns[Stats]()
+
+	cols.MustSetExtractor("ip", func(stats *Stats) (ret string) {
+		if stats.Family == syscall.AF_INET {
+			return "4"
+		}
+		return "6"
+	})
+	cols.MustSetExtractor("sent", func(stats *Stats) (ret string) {
+		return fmt.Sprintf("%d", stats.Sent/1024)
```

Comment:
Use also `units.BytesSize`? (It can be done in a different PR as it's not strictly related to this)